database/core_templates/snapshot.json

```diff
@@ -11087,13 +11087,13 @@
     {
       "applicationResponseId": null,
       "applicationSerial": null,
-      "filePath": "/fda_CylhAzxRhSX_QjtArq3bi.png",
+      "filePath": "fda_CylhAzxRhSX_QjtArq3bi.png",
```

Comment:
Yeah it works. Not sure how it ended up like this. All new files uploaded to the root files folder get entered like this in the database. I think both work, if it's at the root level.

(I only changed this because I thought it was the cause of the logo not showing up on initial load, but turns out it wasn't)